src/mumble/mumble_en.ts

```diff
@@ -20,19 +20,19 @@
         <translation type="unfinished"></translation>
     </message>
```

Comment:
Current problems:

OS - operating system
Operating system settings - system settings
on/off - turn on / turn off
enable disable - turn on / off
One panel of wizard says "positional audio" up top instead of explanation as in others.
"attenuate" in use
"Invalid channel" - why? could it be more precise.